src/requestCertificate.ts

```diff
@@ -24,11 +24,12 @@ export interface UserCsr {
   pkcs10: CertData
 }
 
-export const createUserCsr = async ({ zbayNickname, commonName, peerId }): Promise<UserCsr> => {
+export const createUserCsr = async ({ zbayNickname, commonName, peerId, publicKey }): Promise<UserCsr> => {
   const pkcs10 = await requestCertificate({
     zbayNickname: zbayNickname,
     commonName: commonName,
     peerId: peerId,
+    publicKey: publicKey,
```

Comment:
PublicKey is not a good name, because we have so many of them already. Let's call it something like dmPublicKey perhaps?